packages/fx-core/templates/core/README-auto-generated.md

```diff
@@ -0,0 +1,77 @@
+# TeamsFx Project
+
+## Project folder structure
```

Comment:
Refer to this page (still in review)
https://review.docs.microsoft.com/en-us/microsoftteams/platform/toolkit/debug-background-process?branch=pr-en-us-5468#take-a-tour-of-your-app-source-code
It is from content writer so description looks more professional :)